code/src/assets/styles/global.css

```diff
@@ -0,0 +1,101 @@
+@import "./fonts.css";
+@import "./variables.css";
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: var(--primaryFont);
+  text-align: center;
+  position: relative;
+  height: 100vh;
+  display: grid;
+  align-items: center;
+  background-image: url("../images/sun.svg");
+  background-color: #e6eff4;
+  background-repeat: no-repeat;
+  background-size: 400px;
+  background-position: left top;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+p {
+  line-height: 26px;
+}
+
+ul,
+li {
+  padding: 0;
+  margin: 0;
+}
+
+textarea:focus,
+input:focus,
+input[type]:focus,
+select:focus {
+  border-color: #f1c21b;
+  outline: 0 none;
+}
+
+[type="radio"]:checked,
+[type="radio"]:not(:checked) {
+  position: absolute;
+  left: -9999px;
+}
+[type="radio"]:checked + label,
+[type="radio"]:not(:checked) + label {
+  position: relative;
+  padding-left: 28px;
+  cursor: pointer;
+  line-height: 20px;
+  display: inline-block;
+  color: #666;
+}
+[type="radio"]:checked + label:before,
+[type="radio"]:not(:checked) + label:before {
+  content: "";
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 18px;
+  height: 18px;
+  border: 1px solid #ddd;
+  border-radius: 100%;
+  background: #fff;
+}
+[type="radio"]:checked + label:after,
+[type="radio"]:not(:checked) + label:after {
+  content: "";
+  width: 12px;
+  height: 12px;
+  background: #f1c21b;
+  position: absolute;
+  top: 4px;
+  left: 4px;
+  border-radius: 100%;
+  -webkit-transition: all 0.2s ease;
+  transition: all 0.2s ease;
+}
+[type="radio"]:not(:checked) + label:after {
+  opacity: 0;
+  -webkit-transform: scale(0);
+  transform: scale(0);
```

Comment:
Maybe an idea, to add some small comments pseudo code to know what the code does. Not everywhere of course,  but maybe where there are some animations etc. Maybe it will be hard to remember everything later when looking back at project, what was the end goal of the animation etc. Just an idea 😄

Reply:
Thanks Sara! definitely! I'll do it :+1: